ClientRuntimes/CSharp/ClientRuntime.Tests/ClientRuntime.Tests.csproj

```diff
@@ -12,14 +12,19 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Rest.ClientRuntime.Tests</RootNamespace>
-    <AssemblyName>Microsoft.Rest.ClientRuntime.Tests</AssemblyName>
+    <AssemblyName>ClientRuntime.Tests</AssemblyName>
     <AutoRestTest>true</AutoRestTest>
     <FileAlignment>512</FileAlignment>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
     <NuGetPackageImportStamp>464ffb68</NuGetPackageImportStamp>
   </PropertyGroup>
-  <Import Project="$(SolutionDir)\tools\AutoRest.Settings.targets" />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'mono|AnyCPU' ">
```

Comment:
please move this to AutoRest.Settings.targets, where you can see we did for other build configs